### DISCUSSION TITLE
G 711 streaming with Arduino GIGA

### DISCUSSION THREAD

**Discussion by edwardruthven** [pschatzmann/arduino-audio-tools#1695] [Q&A] [answered]
Hi, I am battling to workout the parameter settings required for Arduino Advanced Analog.

// Resolution, sample rate, number of samples per channel, queue depth.
    if (!adc1.begin(AN_RESOLUTION_16, 8000, 32, 64)) {
        Serial.println("Failed to start analog acquisition!");
        while (1);
    }

The Resolution values available is 
AN_RESOLUTION_8
AN_RESOLUTION_10
AN_RESOLUTION_12
AN_RESOLUTION_14
AN_RESOLUTION_16

All I know is Ulaw is 14 bit signed and Alaw is 13 bit signed - so what do I use from above.
Also what should I use for samples per channel and queue depth?

**Comment by pschatzmann** [comment, marked as answer]
You are completely wrong here: I think you wanted to [post this here](https://github.com/arduino-libraries/Arduino_AdvancedAnalog)
